kuma/static/js/contribution-handler.js

```diff
@@ -217,8 +217,8 @@
 
     var isPopoverBanner = $('.contribution-banner').hasClass('contribution-popover');
 
-    /* If `isPopoverBanner` is false, then this is the
-       contribute page. Init the handler immediately */
+    // If `isPopoverBanner` is false then this is the contribute page.
+    // Init the handler immediately
```

Comment:
We follow JSDoc comment style. The standard for multi line comments that are not documenting a function is to use the following style:

```
/* If `isPopoverBanner` is false, then this is the
   contribute page. Init the handler immediately */
```